app/controllers/peeps.rb

```diff
@@ -0,0 +1,34 @@
+module Chitter
+  module Routes
+    class Peeps < Base
+
+      get '/' do
+        redirect('/peeps')
+      end
+
+      get '/peeps' do
+        @peeps = Peep.all
+        erb :'peeps/index'
+      end
+
+      get '/peeps/new' do
+        unless current_user
```

Comment:
Style/UnlessElse: Do not use `unless` with `else`. Rewrite these with the positive case first.